presto-docs/src/main/sphinx/optimizer/push-down.rst

```diff
@@ -0,0 +1,140 @@
+=========
+Push down
+=========
+
+Presto can push down processing queries, or segments of queries, down to the
+connected data source. This means that a specific function, or other operation,
+is passed through to the underlying database for processing.
+
+The results of this push down can include the following benefits:
+
+* improved overall query performance
+* reduced network traffic between Presto and the database
+* reduced load on the database
+
+Support for push down is specific to each connector and the relevant underlying
+database or storage system.
+
+Every Presto aggregate function is eligible for push down, as long as it
+operates on types supported by the given connector.
```

Comment:
This isn't really correct. Any aggregate that operates directly on a connector column is eligible for pushdown. As a consequence, it would have to be on a type supported by the connector, but that's not the specific reason it is eligible. The function also has to have the same semantics (or the connector might choose to ignore those differences ... such as numeric overflow).

Probably just remove this sentence.

Reply:
Removed.. I just got that info from @findepi originally.